src/org/omegat/gui/preferences/view/SaveOptionsPanel.java

```diff
@@ -56,7 +56,6 @@ private void initComponents() {
         java.awt.GridBagConstraints gridBagConstraints;
 
         intervalDescriptionPanel = new javax.swing.JPanel();
-        intervalDescriptionTextArea = new javax.swing.JTextArea();
```

Comment:
You should not directly edit `SaveOptionsPanel.java` but you edit it through `NetBeans` GUI editor.
The change break `SaveOptionsPanel.form` divert with generated Java code.

Reply:
It builds properly here. I'm OK with editing things in Netbeans but I asked how to use the GUI Editor in the past and nobody answered.

Reply:
I remember a question was too general, and I could not say [RTBM](https://netbeans.apache.org/tutorial/main/kb/docs/java/quickstart-gui/) in public ML for core team.

But I need to say readthedocs of [Coding Styles](https://omegat.readthedocs.io/en/latest/30.CodingStyles/) which can be improved to explain clearer.

Reply:
I don’t know if that's the information I need, but that's the kind of link you could have posted when I asked for information.

Regarding the coding style file, we may need to improve it a bit but I need first to identify the reason why I was stuck.

Thank you for the reference.